backend/migrations/versions/2258cd8d6e9f_resource_treshholds_added.py

```diff
@@ -0,0 +1,37 @@
+"""describe_changes_shortly
```

Comment:
nit: The name given in line 1 is the name that appears when we use alembic CLI commands, can we rename it to match the file name?